include/envoy/network/filter.h

```diff
@@ -81,8 +81,7 @@ class ReadFilter {
   /**
    * Called when data is read on the connection.
    * @param data supplies the read data which may be modified.
-   * @param end_stream supplies whether this is the last byte on the connection. This will only
-   *        be set if the connection has half-close semantics enabled.
+   * @param end_stream supplies whether this is the last byte on the connection.
```

Comment:
Can you make this comment a bit more clear? end_stream may not be set on the last byte in the connection, if the FIN arrives after the last byte on a split packet.

Reply:
Done